src/main/java/racingcar/io/output/ConsoleWriter.java

```diff
@@ -0,0 +1,64 @@
+package racingcar.io.output;
+
+import racingcar.domain.car.Car;
+import racingcar.domain.car.Cars;
+import racingcar.domain.judge.WinnerCars;
+import racingcar.global.CharacterSymbol;
+
+public class ConsoleWriter {
+
+    private static final String MOVE_RESULT_FORMAT = "%s : %s";
+    private static final String WINNER_RESULT_FORMAT = "최종 우승자 : %s";
+
+    public void printCarNameInputGuide() {
+        printLine(OutputMessage.CAR_NAME_INPUT_GUIDE.getLiteral());
+    }
+
+    public void printMovingCountInputGuide() {
+        printLine(OutputMessage.MOVING_COUNT_INPUT_GUIDE.getLiteral());
+    }
+
+    public void printExecutionResultMessage() {
+        printEmptyLine();
+        printLine(OutputMessage.EXECUTION_RESULT.getLiteral());
+    }
+
+    public void printMoveResult(Cars cars) {
+        cars.getCars().forEach(car -> printLine(toMoveResultFormat(car)));
+        printEmptyLine();
+    }
```

Comment:
I/O는 작업은 비용이 큰 작업으로 알고 있습니다
반복문을 통해 매번 이동 결과를 출력하는 것 보다 출력 결과를 모아서 한 번에 출력하는 것은 어떨지 의견 드립니다!
```suggestion
    public void printMoveResult(Cars cars) {
        StringBuilder stringBuilder = new StringBuilder();
        cars.getCars().forEach(car -> stringBuilder.append(toMoveResultFormat(car)).append("\n"));
        printLine(stringBuilder.toString());
    }
```

Reply:
확실히 출력 결과를 한 번에 모아서 출력해주나, 반복하며 출력하나 출력 결과에는 차이가 없을 것 같네요. 저 역시 이 방법이 더 좋은 것 같습니다! 인사이트 감사합니닷 🤠